vite.config.js

```diff
@@ -1,15 +1,13 @@
 import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill"
 import { defineConfig } from "vite"
-import elmPlugin from "vite-plugin-elm"
-
+// import elmPlugin from "vite-plugin-elm"
```

Comment:
If we decide there's nothing we still need from the `elm` files, I can remove this line, as well as the `vite-plugin-elm` package

Reply:
You can remove all Elm files 👍